### ISSUE TITLE
Spotless 6.0 never seems to complete on Android project

### ISSUE THREAD

**Issue by j-roskopf** [diffplug/spotless#994]
Summary of problem:

After adding spotless to my Android project using version 6.0.0, it never seems to complete. I let `gradlew spotlessApply` run for over 20 minutes and I didn't see anything move. It was stuck at 8% for the entire time. 

Gradle version:

7.0.2

Spotless version:

6.0.0

OS and Version:
Edition	Windows 10 Home
Version	20H2
OS build	19042.1288

Spotless configuration block:
```
apply plugin: "com.diffplug.spotless"

spotless {
    java {
        target '**/*.java'
        googleJavaFormat().aosp()
        removeUnusedImports()
        trimTrailingWhitespace()
        indentWithSpaces()
        endWithNewline()
    }

    kotlin {
        target '**/*.kt'
        ktlint("0.41.0")
        endWithNewline()
    }

    kotlinGradle {
        target '*.gradle.kts'
        ktlint()
    }
}
```

It's worth noting, if I downgrade to `classpath "com.diffplug.spotless:spotless-plugin-gradle:5.14.3"`, it works as I would expect

**Comment by nedtwigg** [comment]
Does Gradle 7.3 fix it?  I wonder if this is a dupe of #918. There is an unexplained interaction between Spotless and Kotlin 1.5 that was resolved in Gradle 7.3...

**Comment by j-roskopf** [comment]
Updating to gradle 7.3 fixed it the issue! I was able to update spotless back to 6.0.0 and have it complete as I'd expect. 

Thanks for the help!
